docs/PacketFence_Checkpoint_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,91 @@
+Checkpoint firewall Quick Integration Guide
+==========================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using the *Checkpoint firewall*.
+
+Assumptions
+-----------
+* You have a configured PacketFence environment with working test equipment;
+* You have a Checkpoint firewall.
+
+
+Quick installation
+------------------
+Step 1: Enabling Identity Awareness on the Security Gateway
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To enable Identity Awareness:
+
+1. Log in to 'SmartDashboard'.
+2. From the 'Network Objects tree', expand the 'Check Point branch'.
+3. Double-click the 'Security Gateway' on which to enable 'Identity Awareness'.
+4. In the 'Software Blades' section, select 'Identity Awareness' on the 'Network Security tab'.
+The 'Identity Awareness Configuration' wizard opens.
+5. Select 'one or more options'. These options set the methods for acquiring identities of managed and
+unmanaged assets.
+6. Select 'AD Query - Lets the Security Gateway seamlessly identify Active Directory users and computers' and click Next.
+The 'Integration With Active Directory' window opens.
+7. Select the Active Directory to configure from the list that shows configured LDAP account units or create a new domain. If you have not set up Active Directory, you need to enter a domain name, username, password and domain controller credentials.
+8. Enter the Active Directory credentials and click Connect to verify the credentials. (Important - For AD Query you must enter domain) administrator credentials.
+9. Click Finish.
+
+Step 2: Enabling RADIUS Accounting on a Security Gateway
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To enable RADIUS Accounting for a Security Gateway:
+1. In the 'SmartDashboard Network Objects tree', open the Security Gateway.
+2. On the 'General Properties' page, make sure that the Identity Awareness Blade is enabled.
```

Comment:
Identity Awareness Blade doesn't exist, it's either Identity Awareness or Switch Blade.